### PR TITLE
Designed printing status pages.

### PR DESCRIPTION
**Pull request by kangyeelim**

### CODE REVIEW

File: src/views/PrintStatus.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  }
+
+  componentDidMount() {
+    //GET NUMBER OF FILES TO BE PRINTED?
+  }
+
+  render() {
+    return (
+            <div>
+                <ThemeProvider theme={theme}>
```
Comment (C-likethis123) [inline src/views/PrintStatus.js]: Is the theme provider necessary?

File: src/views/PrintStatus.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                    <Paper>
+                        <AppBar position="static" alignitems="center" color="primary" style={{ marginBottom: '20px' }}>
+                                <Grid container justify="center" wrap="wrap">
+                                    <Grid item>
+                                        <Typography variant="h4">View Printing Status</Typography>
+                                    </Grid>
+                                </Grid>
+                        </AppBar>
+                        {!this.state.uploading && !this.state.printing && <Grid container justify="center" wrap="wrap">
```
Comment (C-likethis123) [inline src/views/PrintStatus.js]: Use destructuring like this:

`const {uploading, printing} = this.state`

Then in this line you can add: `{!uploading && !printing && <Grid...`

**Review by C-likethis123 — CHANGES_REQUESTED** [review]

File: src/views/PrintStatus.js (hunk 2 above)
Comment (kangyeelim) [reply src/views/PrintStatus.js]: Done

File: src/views/PrintStatus.js (hunk 1 above)
Comment (kangyeelim) [reply src/views/PrintStatus.js]: removed

**Pull request merged**